class-lifterlms-rest-api.php

```diff
@@ -196,15 +196,16 @@ public function rest_api_controllers_init() {
 	 * @since 1.0.0-beta.8 Load webhooks actions a little bit later: at init 6 instead of init 10,
 	 *                     just after all the db tables are created (init 5),
 	 *                     to avoid PHP warnings on first plugin activation.
+	 * @since [version] Bump minimum required version to 6.0.0-alpha.1.
 	 *
 	 * @return void
 	 */
 	public function init() {
 
-		// only load if we have the minimum LifterLMS version installed & activated.
-		if ( function_exists( 'LLMS' ) && version_compare( '3.32.0', LLMS()->version, '<=' ) ) {
+		// Only load if we have the minimum LifterLMS version installed & activated.
+		if ( function_exists( 'LLMS' ) && version_compare( '6.0.0-alpha.1', LLMS()->version, '<=' ) ) {
```

Comment:
While we're editing the line let's switch off the deprecated `LLMS()` function. It has been *officially* deprecated since version 4.0.0 but I haven't added a deprecation warning or staged it for removal because it's so prevalent across our add-ons. Instead I've just been updating it when I notice it near other areas I'm editing.

```suggestion
		if ( function_exists( 'llms' ) && version_compare( '6.0.0-alpha.1', llms()->version, '<=' ) ) {
```